### PR TITLE
ocamlc: add page

### PR DESCRIPTION
**Pull request by mira01**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/ocamlc.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ocamlc
+
+> The OCaml bytecode compiler.
+> Produces executables runnable by OCaml interpreter.
+
+- Create named executable from source file:
+
+`ocamlc -o {{output_name}} {{source_file.ml}}`
```
Comment (sbrl) [inline pages/common/ocamlc.md]: Same as your other PR. Perhaps we could tweak this to make it more consistent with other pages?
```suggestion
`ocamlc -o {{path/to/output}} {{path/to/source_file.ml}}`
```

 - Are there any other examples we could include?
 - Is there any official documentation we can link to?

Comment (agnivade) [reply pages/common/ocamlc.md]: Also, we should include the default usage of `ocamlc` without any flags. And let's name path/to/output as path/to/binary.

**Comment by zdroid** [comment]
@agnivade @sbrl Can I implement your suggestion and merge manually?

P.S. `ocamlc` without any flags is used similarly to g++, it just compiles a file with a predefined name of the output file.

**Comment by agnivade** [comment]
Yes, go ahead.

File: pages/common/ocamlc.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# ocamlc
+
+> The OCaml bytecode compiler.
+> Produces executables runnable by OCaml interpreter.
+
+- Create named executable from source file:
+
+`ocamlc -o {{path/to/binary}} {{path/to/source_file.ml}}`
+
+- Create automatically named executable from source file:
```
Comment (agnivade) [inline pages/common/ocamlc.md]: The "automatically named executable" part is not super-clear to me. I get that the name of the executable is decided by ocamlc, but can we reword it to make it clearer ?

And also, let's change executable to binary to be consistent with the examples.

Comment (zdroid) [reply pages/common/ocamlc.md]: Check it out now, it should be much more clear.

Comment (agnivade) [reply pages/common/ocamlc.md]: Much better now. But I'm curious to know what is the final name of the binary that gets created ?
